portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,56 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Jie Liang's Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
```

Comment:
@jieliangang, I have informational comment to improve performance better. See if you can read and understand the article below.
https://flaviocopes.com/javascript-async-defer/

Reply:
Noted with thanks! I have fixed my code and improved the script loading performance. I’ll also spend more time to learn more about DOM.

I have just finalised my portfolio website for this week, and deployed it on http://jang-sps-summer20.appspot.com/.

Reply:
You can click "Resolve conversation" if the comment has been resolved.